js/components/Font.js

```diff
@@ -0,0 +1,11 @@
+const React = require("react");
+
+class Font extends React.Component {
+	render() {
+		return (
+			<Font style = {{color: 'blue'}}/>
```

Comment:
Also, you do **not** want to use `<Font>` here. The Font component can't render itself. You want lowercase `<font>`